pytest/tests/mocknet/mirror.py

```diff
@@ -451,6 +468,14 @@ def run_env_cmd(args, traffic_generator, nodes):
     parser.add_argument('--start-height', type=int)
     parser.add_argument('--unique-id', type=str)
     parser.add_argument('--local-test', action='store_true')
+    parser.add_argument('--type',
+                        type=str,
+                        choices=['all', 'nodes', 'traffic'],
```

Comment:
can we add dumper to this list? or is dumper included in the 'nodes' list?

Reply:
correct, dumper is in nodes